examples/custom-resources/externalname-services/transport-server/externalname-svc.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: externalname-service
+spec:
+  type: ExternalName
+  externalName: external-backend-svc.external-ns.svc.cluster.local
```

Comment:
```suggestion
  externalName: secure-app-external-backend-svc.external-ns.svc.cluster.local
```

Reply:
This still needs to be changed - I ran through the example again just now and I hit the same issue where I had to change this line myself locally

Reply:
Fixed